war/src/main/java/executable/Main.java

```diff
@@ -401,7 +381,7 @@ private static String readStringNonBlocking(InputStream in, int maxToRead) {
     }
 
     private static void trimOffOurOptions(List<String> arguments) {
-        arguments.removeIf(arg -> arg.startsWith("--daemon") || arg.startsWith("--logfile") || arg.startsWith("--extractedFilesFolder")
+        arguments.removeIf(arg -> arg.startsWith("--extractedFilesFolder")
```

Comment:
The removal of the `--daemon` option here is unrelated code cleanup. I removed that option a long time ago but forgot to remove it from this little bit of code.

Reply:
> we have some legacy systems still runnning with an old centOS 6

Jenkins does not support CentOS 6.